config/routes.rb

```diff
@@ -5,7 +5,7 @@
 
   resources :teachers, except: [:index] do
     scope module: :teachers do
-      resources :students, only: [:index]
+      resources :students, only: [:index, :show]
```

Comment:
take a look at the comments I made on teachers controller actions, we may need some new actions here for students

Reply:
You're right, Now I need all crud functions